api/rs/slint/lib.rs

```diff
@@ -171,23 +171,23 @@ The follow table summarizes the entire mapping:
 
 | `.slint` Type | Rust Type | Note |
 | --- | --- | --- |
-| `int` | `i32` | |
-| `float` | `f32` | |
+| `angle` | `f32` | The value in degrees |
+| anonymous object | anonymous tuple | The fields are in alphabetical order. |
+| `array` | [`ModelRc`] | Arrays are represented as models, so that their contents can change dynamically. |
 | `bool` | `bool` | |
-| `string` | [`SharedString`] | A reference-counted string type that can be easily converted to a str reference. |
-| `color` | [`Color`] | |
 | `brush` | [`Brush`] | |
+| `color` | [`Color`] | |
+| `duration` | `i64` | At run-time, durations are always represented as signed 64-bit integers with millisecond precision. |
+| enumeration | `enum` of the same name | The values are converted to CamelCase |
+| `float` | `f32` | |
 | `image` | [`Image`] | |
-| `physical_length` | `f32` | The unit are physical pixels. |
+| `int` | `i32` | |
 | `length` | `f32` | At run-time, logical lengths are automatically translated to physical pixels using the device pixel ratio. |
-| `duration` | `i64` | At run-time, durations are always represented as signed 64-bit integers with millisecond precision. |
-| `angle` | `f32` | The value in degrees |
+| `physical_length` | `f32` | The unit are physical pixels. |
+| `Point` | [`LogicalPosition`] | A struct with `x` and `y` fields, representing logical coordinates. |
 | `relative-font-size` | `f32` | Relative font size factor that is multiplied with the `Window.default-font-size` and can be converted to a `length`. |
+| `string` | [`SharedString`] | A reference-counted string type that can be easily converted to a str reference. |
 | structure | `struct` of the same name | |
-| anonymous object | anonymous tuple | The fields are in alphabetical order. |
-| enumeration | `enum` of the same name | The values are converted to CamelCase |
-| array | [`ModelRc`] | Arrays are represented as models, so that their contents can change dynamically. |
```

Comment:
These are not  literal value and it makes sense to keep them separate at the end IMHO